feature-libs/order/components/order-details/order-details-extended/download-invoices/download-order-invoices-dialog.component.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Component,
+  ChangeDetectionStrategy,
+  OnInit,
+  OnDestroy,
+  ViewChild,
+  ChangeDetectorRef,
+  AfterViewChecked,
+  inject,
+} from '@angular/core';
+import { Order, OrderOutlets } from '@spartacus/order/root';
+import { InvoicesListComponent } from '@spartacus/pdf-invoices/components';
+import { PDFInvoicesFacade } from '@spartacus/pdf-invoices/root';
+import {
+  ICON_TYPE,
+  FocusConfig,
+  LaunchDialogService,
+} from '@spartacus/storefront';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-download-order-invoices-dialog',
+  templateUrl: './download-order-invoices-dialog.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DownloadOrderInvoicesDialogComponent
```

Comment:
[MEDIUM]
Let's stick to the convention of synchronizing the component's: selector, class name and filename (and component module name).
Currently we have:
```
DownloadOrderInvoicesDialogComponent
cx-download-invoices-dialog
download-order-invoices-dialog.component.ts
```

Note: please review all the components in this PR and make sure the names are consistent

Reply:
done